common/oauthTokenManager.go

```diff
@@ -84,7 +84,7 @@ func newAzcopyHTTPClient() *http.Client {
 				Timeout:   10 * time.Second,
 				KeepAlive: 10 * time.Second,
 				DualStack: true,
-			}).Dial,                   /*Context*/
```

Comment:
This file does not have a code change so may be you can sync this up with main and keep the file as is.

Reply:
done